### PR TITLE
fix: Change "Question" to "Lesson" The Odin Project

### PR DESCRIPTION
**Pull request by Sembauke**
- **fix: change "question" to "lesson"**
- **fix: change dashedname**
- **fix: change title**
- **fix: metas**

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

blocked until #54409 and #55027 are merged

### CODE REVIEW

File: curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-lesson-d.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-title: Introduction to Flexbox Question D
+title: Introduction to Flexbox Lesson D
 challengeType: 15
-dashedName: introduction-flexbox-question-d
+dashedName: introduction-flexbox-lesson-d
 ---
 # --description--
 
-Now that you have a basic understanding of how flexbox works, what is the primairy role of a flex item in a `flexbox` layout? Try and answer the question below.
+Now that you have a basic understanding of how flexbox works, what is the primairy role of a flex item in a `flexbox` layout? Try and answer the lesson below.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-lesson-d.md:9]: here wouldn't fit better `question`?

Comment (Sembauke) [reply curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-lesson-d.md:9]: naaaah

Comment (lasjorg) [reply curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-lesson-d.md:9]: So this should be reverted to `question`, correct?

It hasn't, so this doesn't seem resolved.

Comment (Sembauke) [reply curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-lesson-d.md:9]: You know what, I got my mind all mixed up with noodles.

**Review by lasjorg — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**